lib/ui/window.dart

```diff
@@ -210,7 +210,19 @@ class Window {
   /// [physicalSize], or [padding] values change, for example when the device is
   /// rotated or when the application is resized (e.g. when showing applications
   /// side-by-side on Android).
-  VoidCallback onMetricsChanged;
+  ///
+  /// The framework invokes this callback in the same zone in which the
+  /// callback was set.
+  VoidCallback get onMetricsChanged => _onMetricsChanged;
+
+  VoidCallback _onMetricsChanged;
+  Zone _onMetricsChangedZone;
+
+  /// Updates the callback.
```

Comment:
That doc is worthless (that's what a setter for a callback does). But could you add that it tracks the zone, or something? Saying that it remembers the current Zone and that the engine will call the callback in that Zone would be useful.